packages/utils/src/mintReceiptNFT.ts

```
@@ -121,9 +122,18 @@ export async function mintAndSaveReceiptNFT({
     // #endregion
 
     // #region: Initialize blockchain tools and verify transaction
-    const chainTool = BlockchainServerInterfaces.evm
-    chainTool.setChain(chain)
-
+    let chainTool
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)